src/Pages/Home.jsx

```diff
@@ -1,11 +1,19 @@
 import { useSelector } from "react-redux";
+import HomeBanner from "../components/HomeBanner/HomeBanner";
 import BlogList from "../components/BlogList/BlogList";
 
 const Home = () => {
+  const { authUser } = useSelector((state) => state.auth);
   const { blogs } = useSelector((state) => state.blogs);
+  const blogsByOtherAuthor = blogs.filter(
+    ({ author }) => author.id !== authUser?.id
+  );
+  const randomBlog =
+    blogsByOtherAuthor[Math.floor(Math.random() * blogsByOtherAuthor.length)];
```

Comment:
please explain the reason for doing this?

maybe a better readable version

```
const numberOfBlogs = blogsByOtherAuthor.length;
const randomIndex = Math.floor(Math.random() * numberOfBlogs);
const randomBlog = blogsByOtherAuthor[randomIndex];
```

Reply:
To show random blog in home banner.